force-app/main/default/lwc/serviceScheduleReview/serviceScheduleReview.js

```diff
@@ -0,0 +1,197 @@
+            SessionStart__c: { apiName: "SessionStart__c", label: "Session Start" },
+            SessionEnd__c: { apiName: "SessionEnd__c", label: "Session End" },
+        },
+    };
+    @api schedule = {
+        record: {
+            Name: "Family Class (Thu-Fri)",
+            PrimaryServiceProvider__c: { Id: "000", Name: "Veronica Mars" },
+            OtherServiceProvider__c: { Id: "001", Name: "Kristen Bell" },
+            ParticipantCapacity__c: 12,
+        },
+        label: "Service Schedule",
+        pluralLabel: "Service Schedules",
+        fields: {
+            FirstSessionStart__c: {
+                apiName: "FirstSessionStart__c",
+                label: "First Session Start",
+            },
+            FirstSessionEnd__c: {
+                apiName: "FirstSessionEnd__c",
+                label: "First Session End",
+            },
+            PrimaryServiceProvider__c: {
+                apiName: "PrimaryServiceProvider__c",
+                label: "Primary Service Provider",
+            },
+            OtherServiceProvider__c: {
+                apiName: "OtherServiceProvider__c",
+                label: "Other Service Provider",
+            },
+            ParticipantCapacity__c: {
+                apiName: "ParticipantCapacity__c",
+                label: "Participant Capacity",
+            },
+        },
+    };
+
+    get reviewLabel() {
+        return format(REVIEW_RECORDS, [this.schedule.label]);
+    }
+
+    get sessionDataTableColumns() {
+        return [
+            {
+                label: this.sessions.fields.Name.label,
+                fieldName: this.sessions.fields.Name.apiName,
+                hideDefaultActions: true,
+            },
+            {
+                label: this.sessions.fields.SessionStart__c.label,
+                fieldName: this.sessions.fields.SessionStart__c.apiName,
+                hideDefaultActions: true,
+            },
+            {
+                label: this.sessions.fields.SessionEnd__c.label,
+                fieldName: this.sessions.fields.SessionEnd__c.apiName,
+                hideDefaultActions: true,
+            },
+        ];
+    }
+
+    get participantDataTableColumns() {
+        return [
+            {
+                label: this.participants.fields.Contact__c.label,
+                fieldName: this.participants.fields.Contact__c.apiName,
+                hideDefaultActions: true,
+            },
+            {
+                label: this.participants.fields.Email.label,
+                fieldName: this.participants.fields.Email.apiName,
+                hideDefaultActions: true,
+            },
+            {
+                label: this.participants.fields.Stage__c.label,
+                fieldName: this.participants.fields.Stage__c.apiName,
+                hideDefaultActions: true,
+            },
+        ];
+    }
+
+    get lastSessionEndDateTime() {
+        return new Date([...this.sessions.records].pop().SessionEnd__c);
```

Comment:
Are we doing all of this because it's hardcoded strings at the moment?

Reply:
I think so. Will revisit once we have real data.